### ISSUE TITLE
ASL distribution no longer includes OpenIconic SVGs

### ISSUE THREAD

**Issue by ahus1** [plantuml/plantuml#1554]
**Describe the bug**
While 1.2023.7 ASL version as downloaded from SourceForge https://sourceforge.net/projects/plantuml/files/1.2023.7/ contains the SVG images for OpenIconic, the version 1.2023.10 ASL downloaded from GitHub releases and Maven Central doesn't. [Looking at the licenses, MIT licensed content can be bundled with ASL AFAIK](https://law.stackexchange.com/questions/6081/can-i-bundle-mit-licensed-components-in-a-apache-2-0-licensed-project#:~:text=Since%20the%20MIT%20license%20is,together%20under%20the%20Apache%20license.).

**To Reproduce**

Given the file: 

```
@startuml
listopeniconic
@enduml
```

```
java -jar plantuml.jar test.iconic -tpng
```

With the JAR from plantuml-jar-asl-1.2023.5.zip it produces a PNG with the list of all icons. 

With the JAR plantuml-asl-1.2023.10.jar it produces the error: 

```
Exception in thread "main" java.lang.NullPointerException
        at java.base/java.io.Reader.<init>(Reader.java:168)
        at java.base/java.io.InputStreamReader.<init>(InputStreamReader.java:76)
        at net.sourceforge.plantuml.openiconic.PSystemListOpenIconic.getRootDrawable(PSystemListOpenIconic.java:36)
        at net.sourceforge.plantuml.PlainDiagram.exportDiagramNow(PlainDiagram.java:31)
        at net.sourceforge.plantuml.AbstractPSystem.exportDiagram(AbstractPSystem.java:163)
        at net.sourceforge.plantuml.PSystemUtils.exportDiagramsDefault(PSystemUtils.java:170)
        at net.sourceforge.plantuml.PSystemUtils.exportDiagrams(PSystemUtils.java:60)
        at net.sourceforge.plantuml.SourceFileReaderAbstract.getGeneratedImages(SourceFileReaderAbstract.java:156)
        at net.sourceforge.plantuml.Run.manageFileInternal(Run.java:476)
        at net.sourceforge.plantuml.Run.processArgs(Run.java:370)
        at net.sourceforge.plantuml.Run.manageAllFiles(Run.java:337)
        at net.sourceforge.plantuml.Run.main(Run.java:172)
```

**Expected behavior**

The list of icons should show. The icons should also show in the salt forms (which they currently don't, as reported in https://github.com/asciidoctor/asciidoctor-intellij-plugin/issues/1446). 

Expected output: 

![test](https://github.com/plantuml/plantuml/assets/3957921/51733c11-c283-48c7-86b9-9f28980202e9)

**Comment by arnaudroques** [comment]
Sorry about that! Our build scripts were not perfect... obviously.
Thanks for the feedback.
Could you check the snapshot release at https://github.com/plantuml/plantuml/releases/tag/snapshot are ok for you?
We will then release a new official version in the incoming days.

**Comment by ahus1** [comment]
Thank you for the quick turnaround. I tested the version and icons work both with `listopeniconic` and when embedded in salt diagrams.

Looking forward to the next release!

**Comment by ahus1** [comment]
I did a comparison of the JARs, and I think I found one more difference around the `graphviz.dat` file, see #1556

**Comment by arnaudroques** [comment]
This should be fixed [in release 1.2023.12](https://github.com/plantuml/plantuml/releases)
Please reopen this issue if it is not working for you!
Thanks !
